tailon/server.py

```diff
@@ -74,6 +75,17 @@ def get(self, check=None):
             message = self.application.file_lister.dirs
         self.write(escape.json_encode(message))
 
+class GetLevels(BaseHandler):
+    def get(self, app=None):
+        levels = {}
+        self.set_header('Cache-Control', 'no-store, no-cache, must-revalidate, max-age=0')
+        self.application.file_lister.refresh()
+        dirs = self.application.file_lister.all_dir_names
```

Comment:
Could you add here the code to import custom headers ? It's useful if we want to use ajax for it

The bit of code is here at lines 52-53 https://github.com/criteo-forks/tailon/blob/v1.3.0-criteo.1/tailon/server.py
`        for header in self.config["http-headers"]:
            self.set_header(header, self.config["http-headers"][header])`